lang/strings_english.txt

```diff
@@ -1187,7 +1187,7 @@ $s_issue_id = 'Issue #';
 $s_recently_visited = 'Recently Visited';
 $s_priority_abbreviation = 'P';
 $s_note_user_id_label = 'Note By:';
-$s_filter_match_type = 'Match Type';
+$s_filter_match_type_label = 'Match Type:';
```

Comment:
Is it worth breaking the translation completely until all languages are translated again?  Why don't we use the same language string?  Another option is to rename it in all languages.